core/cmd/vrf_keys_commands_test.go

```diff
@@ -6,14 +6,16 @@ import (
 	"os"
 	"testing"
 
+	"github.com/urfave/cli"
+
 	"github.com/smartcontractkit/chainlink/core/utils"
 	"github.com/smartcontractkit/chainlink/core/web/presenters"
-	"github.com/urfave/cli"
 
-	"github.com/smartcontractkit/chainlink/core/cmd"
-	"github.com/smartcontractkit/chainlink/core/internal/cltest"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
```

Comment:
nit/ this one still has two split groups of `github.com/smartcontractkit/chainlink/core/`

Reply:
Hmm... look like it's all in one block to me.  I think goimports does that automatically.

Reply:
Oh sorry, was looking at the wrong file!  Weird